sdk.mjs

```diff
@@ -175,10 +175,8 @@ export const normalizeNewlines = (text) => {
 export const normalizeColor = (text) => {
   let color = text.replace('#', '').toUpperCase();
   if (color.length < 6) {
-    color = color
-      .slice(0, 3)
-      .map((x) => x.repeat(2))
-      .join('');
+    const firstThreeColorChars = color.slice(0, 3);
+    color = [...firstThreeColorChars].map((x) => x.repeat(2)).join('');
```

Comment:
```suggestion
    // eslint-disable-next-line unicorn/no-useless-spread
    color = [...color.slice(0, 3)].map((x) => x.repeat(2)).join('');
```

Thank you for contributing. We can ignore the `unicorn/no-useless-spread` error here.

Reply:
Thanks! 😊  Done in 83f676fd09ae321346e778158c8758ba95d4ae61